doc/modules/mod_inbox.md

```diff
@@ -79,6 +79,26 @@ Server:
 </iq>
 ```
 
+### Reseting inbox
+
+An inbox can be reset at will using the stanza
+
+```xml
+<iq type='set'>
+    <reset xmlns='erlang-solutions.com:xmpp:inbox:0#conversation' jid='interlocutor_bare_jid'/>
+</iq>
+```
+
+Where `jid` is the bare jid of the interlocutor whose inbox wants to be reset.
+This action does not change the last message stored in inbox; that is, nor this
```

Comment:
This action does not change the last message stored in inbox; meaning that neither this